crates/eip7594/src/trusted_setup.rs

```diff
@@ -6,6 +6,12 @@ use crate::constants::{FIELD_ELEMENTS_PER_BLOB, FIELD_ELEMENTS_PER_CELL};
 
 const TRUSTED_SETUP_JSON: &str = include_str!("../data/trusted_setup_4096.json");
 
+/// Represents the Ethereum trusted setup used for KZG commitments on the BLS12-381 curve.
```

Comment:
```suggestion
/// Represents an Ethereum trusted setup used for KZG commitments on the BLS12-381 curve.
```

We have three or four trusted setups, so just a small nit